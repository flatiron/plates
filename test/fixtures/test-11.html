<input type="hidden" name="method" value="" /><input type="hidden" name="id" value="" />
