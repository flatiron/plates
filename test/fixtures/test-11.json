{
  "method": "DELETE",
  "id": 7
}
