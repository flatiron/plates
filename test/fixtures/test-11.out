<input type="hidden" name="method" value="DELETE" /><input type="hidden" name="id" value="7" />
